draft-ietf-quic-transport.md

```diff
@@ -1007,11 +1007,12 @@ initial connection ID.
 When an endpoint issues a connection ID, it MUST accept packets that carry this
 connection ID for the duration of the connection or until its peer invalidates
 the connection ID via a RETIRE_CONNECTION_ID frame
-({{frame-retire-connection-id}}).  Connection IDs that are issued and not
-retired are considered active; any active connection ID is valid for use with
-the current connection at any time, in any packet type.  This includes the
-connection ID issued by the server via the preferred_address transport
-parameter.
+({{frame-retire-connection-id}}) or it invalidates the connection ID by
+increasing the the Retire Prior To field.  Connection IDs that are issued and
```

Comment:
If the intent of this PR is to say "I'm retiring these connection IDs, whether you like it or not", then maybe a name change is in order.  As this is a declaration, maybe "Retired Prior To" or "Invalid Prior To".

Reply:
I like the "Retired Prior To". I think we should still consider this as retiring the CID, and not introduce a new term/concept like invalid.